pom.xml

```diff
@@ -33,6 +33,7 @@
   <properties>
     <jenkins.version>1.642.4</jenkins.version>
     <java.level>7</java.level>
+    <jenkins-test-harness.version>2.30-20171010.153224-1</jenkins-test-harness.version> <!-- TODO https://github.com/jenkinsci/jenkins-test-harness/pull/78 -->
```

Comment:
I would propose to release JTH and Plugin POM before merging this PR

Reply:
yes, would do

Reply:
Well, JTH at least. `plugin-pom` can come along later, not urgent.